.github/workflows/ci.yml

```
@@ -0,0 +1,25 @@
+---
+name: CI
+
+# Controls when the action will run.
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+  release:
+jobs:
+  tox:
+    runs-on: ubuntu-20.04
+    strategy:
+      fail-fast: false
+      matrix:
+        make_target: [test]
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v2
+        with:
+          python-version: 3.6
+      # GHA won't setup tox for us and we use tox-pip-extensions for venv-update
+      - run: pip install tox==3.2 tox-pip-extensions==1.3.0
+      - run: make ${{ matrix.make_target }}
```

```suggestion
    steps:
      - uses: actions/checkout@v2
      - uses: actions/setup-python@v2
        with:
          python-version: 3.6
      # GHA won't setup tox for us and we use tox-pip-extensions for venv-update
      - run: pip install tox==3.2 tox-pip-extensions==1.3.0
      - run: make test
```

is probably fine for now since we're only running one thing (seems like an unnecessary layer to have a matrix of one target)